applications/callflow/doc/notification.md

```diff
@@ -10,11 +10,14 @@ Validator for the customer defined notification callflow's data object
 
 Key | Description | Type | Default | Required | Support Level
 --- | ----------- | ---- | ------- | -------- | -------------
+`comments` | Text message that need to include into notification | `string()` |   | `false` |  
+`notification_media.[]` |   | `string('email' | 'sms')` |   | `false` |  
+`notification_media` | What is media need to use for notification | `array(string('email' | 'sms'))` |   | `false` |  
```

Comment:
Why validator wants two string definition  for `notification_media` rather one string?
How i can add description for  `notification_media.[]`?